### ISSUE TITLE
False Positive on proto-google-iam-v1-1.0.1.jar

### ISSUE THREAD

**Issue by stefanscheidt** [dependency-check/DependencyCheck#2992]
False positive on library proto-google-iam-v1-1.0.1.jar - reported as `cpe:2.3:a:grpc:grpc:1.0.1:*:*:*:*:*:*:*`
```xml
<dependency>
      <groupId>com.google.api.grpc</groupId>
      <artifactId>proto-google-iam-v1</artifactId>
      <version>1.0.1</version>
</dependency>
```

**Comment by jeremylong** [comment]
included in fix for #2991.  In the future - you can report all of these related FP in a single ticket.

**Comment by stefanscheidt** [comment]
Aye, Aye, Sir!
